unison-src/transcripts/idempotent/fix-4536.md

```diff
@@ -0,0 +1,154 @@
+``` ucm
+scratch/main> builtins.merge lib.builtin
+
+  Done.
+```
+
+``` unison :error
+foo : Nat
+foo =
+  use Nat .+
+  1 + 2
+```
+
+``` ucm :added-by-ucm
+  Loading changes detected in scratch.u.
+
+  I got confused here:
+
+      3 |   use Nat .+
+
+
+  I was surprised to find a '.' here.
+  I was expecting one of these instead:
+
+  * bang
+  * binding
+  * do
+  * false
+  * force
+  * handle
+  * if
+  * lambda
+  * let
+  * newline or semicolon
+  * pattern
+  * quote
+  * termLink
+  * true
+  * tuple
+  * typeLink
```

Comment:
Side note, I'm not sure which one of these it wants.

Reply:
Yeah I know, the error messages are atrocious :|